tests/sentry/api/endpoints/test_group_integration_details.py

```diff
@@ -99,7 +116,7 @@ def test_simple_get_create(self):
                         'required': True,
                     }, {
                         'default': ('Sentry Issue: [%s](%s)\n\n```\n'
-                                    'Stacktrace (most recent call last):\n\n  '
+                                    'Stacktrace (most recent call first):\n\n  '
```

Comment:
Why does this change?

Reply:
I'm not actually sure, @tkaemming - do you know?

Reply:
Hm... seems like that comes [from here](https://github.com/getsentry/sentry/blob/78b51778f1f9671bdea480b08781a174ec81fe91/src/sentry/interfaces/stacktrace.py#L98-L112.) Seems like the changes from `create_group`/`create_event` maybe triggered something like the platform to be saved differently? (I don't know if that's the case, I didn't follow the fixture creation down to the source.)